freedomofpress/securedrop
# 3678 remove apptest browser installs

## Status

Ready for review 

## Description of Changes

Fixes #3678 
This removes xvfb, firefox, and tor-browser installs from app-test role, and therefore from app-staging VM. Functional tests are run remotely so they're no longer necessary. It depends on @kushaldas' changes in #3692 ,so that should be merged first.

## Testing

- Bring up staging environment. 
- Verify that xvfb, firefox, and tbb not installed. Set up test database with single admin account
- Update `securedrop/tests/functional/instance_information.json` with staging instance data
- Run functional tests:
```
cd securedrop
./bin/dev-shell ./bin/run-test tests/functional
```

## Checklist

### If you made changes to the server application code:

- [x ] Linting (`make ci-lint`) and tests (`make -C securedrop test`) pass in the development container
